triagebot.toml

```diff
@@ -408,9 +408,14 @@ trigger_files = [
 trigger_files = [
     "triagebot.toml",
     "rustfmt.toml",
-    "LICENSES",
```

Comment:
There's a folder called `LICENSES`. Certainly adding *more* files here is probably fine, but we probably shouldn't remove that one.

Reply:
Oh, sorry, I didn't see it. I just added it and changed the commit message as well. Thanks!